nighthawk/test/options_test.cc

```diff
@@ -24,6 +26,14 @@ class OptionsImplTest : public testing::Test {
     return std::make_unique<OptionsImpl>(argv.size(), argv.data());
   }
 
+  // We suppress stderr because errors send by tclap mess up the google test output.
+  static void SetUpTestSuite() {
```

Comment:
Is this for debug or permanent? What errors do we get from `tclap`?

Reply:
This was to work around a specific issue to vscode's terminal rendering getting confused. I removed it,  regular terminals don't have that problem. `tclap` reacts to bad input by sending help and complaints to stderr, which the tests do extensively.

Reply:
Is there a way to redirect its logs to Envoy's? We do this in some places, e.g. with nghttp2, see https://github.com/envoyproxy/envoy/blob/1f9e2fd690e3747017f9e7aa8e6368592f5c71e7/source/common/http/http2/codec_impl.cc#L41

Reply:
It looks like that doing that would be pretty straightforward:

1. Implement this interface: https://github.com/maddouri/tclap/blob/1e1cc4fb9abbc4bfcd62c73085c3c446fca681dd/include/tclap/CmdLineOutput.h
2. Use that implementation by calling `setOutput()` on the `TCLAP::CmdLine` instance.